src/common/utils/utils.ts

```diff
@@ -78,7 +78,8 @@ const formatUSD = (amount: number, options?: { compact?: boolean }): string => {
   const { format } = new Intl.NumberFormat(undefined, {
     style: 'currency',
     currency: 'USD',
-    notation: options?.compact ? getFormatUSDNotation(amount) : undefined
+    notation: options?.compact ? getFormatUSDNotation(amount) : undefined,
+    minimumFractionDigits: amount > 0 && amount < 0.01 ? 3 : undefined
```

Comment:
My only comment was whether we could avoid repeating this condition, but since the second instance is in the component library (and will end up in another repo) it's not a blocker.

@danielsimao not specific to this PR, but I'll add a note to the component library ticket about cases like this, where we may have utilities that we need to keep in sync (although this might be an edge case).